compiler/ras/Tree.cpp

```diff
@@ -898,8 +898,8 @@ TR_Debug::printIRTrees(TR::FILE *pOutFile, const char * title, TR::ResolvedMetho
 
          trfprintf(pOutFile, "%s\n",
             _comp->compileRelocatableCode() ?
-                       ((TR_AOTMethodInfo *)ics._methodInfo)->resolvedMethod->signature(comp()->trMemory(), heapAlloc) :
-                       fe()->sampleSignature(ics._methodInfo, 0, 0, _comp->trMemory()));
+                       meth->signature(comp()->trMemory(), heapAlloc) :
+                       fe()->sampleSignature(ics._method, 0, 0, _comp->trMemory()));
```

Comment:
You can just use `fe()->sampleSignature(ics._method, 0, 0, _comp->trMemory())` here; I think the reason for the `?` operator was because `_methodInfo` wasn't be a `TR_OpaqueMethodBlock *` prior to this change; better to just be consistent with non-AOT compilations.